### DISCUSSION TITLE
[REQUEST] Seller Central Report: Unified Transaction Report

### DISCUSSION THREAD

**Discussion by ktroxler** [amzn/selling-partner-api-models#607] [Ideas]
Hello All,

For Seller Central reports it's currently not possible to pull the Unified Transaction Report from Selling Partner API. Is this going to be a future release? Currently 3P reporting for clients is missing a great of metrics from the current version of the API
 
<img width="948" alt="Screen Shot 2022-10-21 at 9 12 07 AM" src="https://user-images.githubusercontent.com/51894654/197203959-f07aec36-21b2-47cb-b682-ecf134eec97a.png">

**Comment by ritikacd** [comment]
Hi @ktroxler
Are you looking to have Data Range Reports available through SP-APIs? if so, may I know how are you consuming this report?

**Reply by ktroxler** [reply]
Yes, would like to have month to date functionally of the Unified Transaction Report that includes Amazon Marketplace, FBA, and Webstore transactions at the product sales and fees level. Currently manually go in and pull these reports, but if we could get via API would be better

**Reply by ritikacd** [reply]
If you are using Date Range reports to get the orders data at sales and fees level, I suggest exploring [Settlement reports](https://developer-docs.amazon.com/sp-api/docs/report-type-values#settlement-reports).  
Flat File Settlement Report(_**GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE**_) includes all orders and adjustments for the settlement period, showing the details of the disbursement and fees to your bank account. But Settlement reports are automatically scheduled by Amazon, they cannot be requested or scheduled. 

Hope this helps!
